src/fusor/fusor.py

```diff
@@ -718,55 +691,22 @@ def add_translated_sequence_id(
                     domain.sequence_location.location.sequence_id = new_id
         return fusion
 
-    def add_gene_descriptor(self, fusion: Fusion) -> Fusion:
-        """Add additional fields to ``gene_descriptor`` in fusion object
-
-        :param fusion: A valid Fusion object
-        :return: Updated fusion with additional fields set in ``gene_descriptor``
-        """
-        properties = [fusion.structural_elements]
-        if fusion.type == FusionType.CATEGORICAL_FUSION:
-            properties.append(fusion.critical_functional_domains)
-
-        for prop in properties:
-            for obj in prop:
-                if "gene_descriptor" in obj.model_fields:
-                    label = obj.gene_descriptor.label
-                    norm_gene_descr, _ = self._normalized_gene_descriptor(
-                        label, use_minimal_gene_descr=False
-                    )
-                    if norm_gene_descr:
-                        obj.gene_descriptor = norm_gene_descr
-        if fusion.regulatory_element and fusion.regulatory_element.associated_gene:
-            reg_el = fusion.regulatory_element
-            label = reg_el.associated_gene.label
-            norm_gene_descr, _ = self._normalized_gene_descriptor(
-                label, use_minimal_gene_descr=False
-            )
-            if norm_gene_descr:
-                reg_el.associated_gene = norm_gene_descr
-        return fusion
-
-    def _normalized_gene_descriptor(
-        self, query: str, use_minimal_gene_descr: bool = True
-    ) -> tuple[GeneDescriptor | None, str | None]:
-        """Return gene descriptor from normalized response.
+    def _normalized_gene(
+        self, query: str, use_minimal_gene_descr: bool
+    ) -> tuple[Gene | None, str | None]:
+        """Return gene from normalized response.
 
         :param query: Gene query
-        :param use_minimal_gene_descr: ``True`` if minimal gene descriptor
```

Comment:
Should add back since `use_minimal_gene_descr` is still being used. Maybe rename to `use_minimal_gene`?